src/main/pages/che-7/administration-guide/proc_workspaces-requirements.adoc

```diff
@@ -60,7 +60,7 @@ The following table presents the default requirements for all workspace componen
 +
 ... Custom memory limit and request:
 +
-If present, the `memoryLimit` attribute of the `containers` section of the `meta.yaml` file defines the memory limit of a `chePlugins` or `cheEditors` component. {prod-short} automatically sets the memory request to match the memory limit. There is no attribute in a `meta.yaml` file to set the memory request.
+If present, the `memoryLimit` and `memoryRequest` attributes of the `containers` section of the `meta.yaml` file defines the memory limit of a `chePlugins` or `cheEditors` component. {prod-short} automatically sets the memory request to match the memory limit in case it is not specified explicitly.
```

Comment:
```suggestion
If present, the `memoryLimit` and `memoryRequest` attributes of the `containers` section of the `meta.yaml` file define the memory limit of the `chePlugins` or `cheEditors` components. {prod-short} automatically sets the memory request to match the memory limit in case it is not specified explicitly.
```